css/style.css

```diff
@@ -1,4 +1,224 @@
 .body {
   font-family: 'Montserrat', sans-serif;
   font-weight: normal;
+  max-width: 1440px;
+  min-width: 320px;
+  overflow-x: hidden;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  padding-top: 24px;
+}
+
+.header__logo-img {
+/* width: 166px; */
+}
+
+.menubar {
+  display: flex;
+}
+
+.menubar__link {
+  text-decoration: none;
+  line-height: 18px;
+  font-weight: 600;
+  font-size: 15px;
+  color: black;
+}
+
+.menubar__link:hover, .menubar__link:active, .menubar__link:focus {
+  cursor: pointer;
+  color: #E51937;
+  box-shadow: 0 2px 0 #E51937;
+  opacity: .75;
+  transition: 0.4s;
+}
+
+.menubar__link_red:nth-child(1) {
+margin-left: 258.14px;
+}
+
+.menubar__link:nth-child(2) {
+  margin-left: 63.14px;
+}
+
+.menubar__link:nth-child(3){
+  margin-left: 24.86px;
+}
+
+.menubar__link:nth-child(4) {
+  margin-left: 44px;
+}
+
+.menubar__link:nth-child(5) {
+  margin-left: 46px;
+}
+
+.menubar__link:nth-child(6) {
+  margin-left: 221.14px;
+}
+
+.menubar__link:nth-child(7) {
+  margin-left: 30px;
+}
+
+.button:nth-child(8) {
+  margin-left: 30px;
+}
+
+.button {
+  display: block;
+  background-color: #ffffff;
+  border: non;
+  width: 24px;
+  height: 15px;
+  position: relative;
+}
+
+.button .button__line, 
+.button .button__line:before,
+.button .button__line:after {
+  position: absolute;
+  display: block;
+  top: 50%; margin-top: -1px;
+  left: 50%; margin-left: -12px;
+  width: inherit;
+  height: 2px;
+  background-color: #E51937;
+}
+
+.button__line:before, .button__line:after {
+  content: '';
+}
+
+.button__line:before {
+ transform: translateY(-6px);
+}
+
+.button__line:after {
+  transform: translateY(6px);
+ }
+
+.button:hover {
+  cursor: pointer;
+  box-shadow: 0 0 5px #473c3c;
+  border: 1px solid #E51937;
+  transition: 0.5s;
+}
+.content {
+  position: relative;
+  /* background: url("../img/jpg/Tesla\ logo\ car.jpg"); */
+}
+
+.content__main-title, .content__title {
+  font-family: Montserrat;
+  font-style: normal;
+  text-align: center;
+}
+
+.content__main-title {
+  position: absolute;
+  font-weight: 600;
+  font-size: 90px;
+  margin: auto;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  padding-top: 60px;
+  z-index: 2;
+}
+
+.content__title {
+  position: absolute;
+  font-weight: 500;
+  font-size: 15px;
+  margin: auto;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  padding-top: 150px;
+  z-index: 2;
+}
+
+.content__back {
+  transform: translateY(40px);
+}
+
+.content__data-car {
+ display: flex;
+ padding-top: 60px;
+ justify-content: space-between;
+ align-items: center;
+ margin-left: 321px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)